TBDataPreparation/202108_SPS/scripts/PhysicsEvent.h

```diff
@@ -52,30 +54,25 @@ PMTCalibration::PMTCalibration(const std::string& fname){
 }
 
 
-class EventOut{
-  public:
-	EventOut(){};
-	~EventOut(){};
-	uint32_t EventID;
-
-        float SPMT1, SPMT2, SPMT3, SPMT4, SPMT5, SPMT6, SPMT7, SPMT8;
-    	float CPMT1, CPMT2, CPMT3, CPMT4, CPMT5, CPMT6, CPMT7, CPMT8;
-        float SiPMPheC[160] = {0};
-        float SiPMPheS[160] = {0};
-        float totSiPMCene = 0.;
-        float totSiPMSene = 0.;
-        float SPMTenergy = 0.;
-        float CPMTenergy = 0.;
-        int PShower, MCounter, C1, C2;
-    	
-    	void CompSPMTene(){SPMTenergy = SPMT1+SPMT2+SPMT3+SPMT4+SPMT5+SPMT6+SPMT7+SPMT8;}
-        void CompCPMTene(){CPMTenergy = CPMT1+CPMT2+CPMT3+CPMT4+CPMT5+CPMT6+CPMT7+CPMT8;}
+struct EventOut{
+    EventOut(){};
+    ~EventOut(){};
+    uint32_t EventID;
+    float SPMT1, SPMT2, SPMT3, SPMT4, SPMT5, SPMT6, SPMT7, SPMT8;
+    float CPMT1, CPMT2, CPMT3, CPMT4, CPMT5, CPMT6, CPMT7, CPMT8;
+    float SiPMPheC[160] = {0};
+    float SiPMPheS[160] = {0};
+    float totSiPMCene = 0.;
+    float totSiPMSene = 0.;
+    float SPMTenergy = 0.;
+    float CPMTenergy = 0.;
+    int PShower, MCounter, C1, C2;
+    inline void CompSPMTene(){SPMTenergy = SPMT1+SPMT2+SPMT3+SPMT4+SPMT5+SPMT6+SPMT7+SPMT8;}
+    inline void CompCPMTene(){CPMTenergy = CPMT1+CPMT2+CPMT3+CPMT4+CPMT5+CPMT6+CPMT7+CPMT8;}
```

Comment:
We should agree on having struct-based output or class-based output. As root examples to store objects are mostly dedicated to classes I would recommend not to changed Event and EventOut from class to struct. Maybe  @ivivarel or @ilnegri7 can comment on this.

Reply:
In C++ a struct is the same as a class with all public members. To me it seemed more logical to use a struct since we are mostly storing data but i doesn't make any difference.

Reply:
Sure, we can technically use structs for this. But in any case, we are going to add getters, setters, and other methods to them, so why not keeping a class? Also, ROOT provides examples storing classes, not sure about structs.